kanidmd/src/lib/core/https/mod.rs

```diff
@@ -1211,10 +448,13 @@ pub fn create_https_server(
             std::process::exit(1);
         }
 
-        tserver.at("/").get(index_view);
-        tserver.at("/ui/").get(index_view);
-        tserver.at("/ui/*").get(index_view);
-        tserver
+        let mut static_tserver = tserver.at("");
```

Comment:
What's going on here? What about it is static?

Reply:
Static web content (IE static files that never change)